grammars/css.cson

```diff
@@ -20,294 +20,1234 @@
+      {
+        # @supports
+        'begin': '(?i)(?=@supports(?:\\s|\\(|$|/\\*))'
+        'end': '(?<=})|(?=;)'
+        'patterns': [
+          {
+            'begin': '\\G'
+            'end': '(?=[;{])'
+            'name': 'meta.at-rule.supports.header.css'
+            'patterns': [
+              {
+               'captures':
+                 '1':
+                   'name': 'punctuation.definition.keyword.css'
+                'match': '(?i)\\G(@)supports'
+                'name': 'keyword.control.at-rule.supports.css'
+              }
+              {
+                'include': '#feature-query-operators'
+              }
+              {
+                'include': '#feature-query'
+              }
+              {
+                'include': '#comment-block'
+              }
+              {
+                'include': '#escapes'
+              }
+            ]
+          }
+          {
+            'begin': '{'
+            'beginCaptures':
+              '0':
+                'name': 'punctuation.section.supports.begin.bracket.curly.css'
+            'end': '}'
+            'endCaptures':
+              '0':
+                'name': 'punctuation.section.supports.end.bracket.curly.css'
+            'name': 'meta.at-rule.supports.body.css'
+            'patterns': [
+              {
+                'include': '$self'
+              }
+            ]
+          }
+        ]
+      }
+      {
+        # @viewport
+        'begin': '(?i)((@)viewport)(?=[\\s{\'";]|/\\*|$)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.viewport.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': '(?=[{;@])'
+        'name': 'meta.at-rule.viewport.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        # @font-feature-values
+        'begin': '(?i)((@)font-feature-values)(?=[\\s{\'";]|/\\*|$)\\s*'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.font-feature-values.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'contentName': 'variable.parameter.font-name.css'
+        'end': '\\s*(?={|[@;])'
+        'name': 'meta.at-rule.font-features.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        # @annotation, @character-variant, @ornaments, @styleset, @stylistic, @swash
+        'include': '#font-features'
+      }
+      {
+        # @namespace
+        'begin': '(?i)((@)namespace)(?=[\\s\'";]|/\\*|$)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.namespace.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': ';|(<=["\')])|(?=[@{])'
```

Comment:
I'm afraid I still don't understand, sorry.  Could you post a code snippet or try some other method of explaining it to me?

I guess right now I'm just stuck on why `@namespace <="` should be matched.  I'm probably missing something simple.